src/meta/mod.rs

```diff
@@ -212,14 +212,15 @@ impl Meta {
         };
 
         #[cfg(unix)]
-        let owner = Owner::from(&metadata);
-        #[cfg(unix)]
-        let permissions = Permissions::from(&metadata);
+        let (owner, permissions) = (Some(Owner::from(&metadata)), Some(Permissions::from(&metadata)));
+
+        #[cfg(all(windows, feature = "no-windows-permissions"))]
+        let (owner, permissions) = (None, None);
 
-        #[cfg(windows)]
+        #[cfg(all(windows, not(feature = "no-windows-permissions")))]
```

Comment:
how about checking this feature inside `windows_utils.rs`, so that we can keep checking `unix/windows` and drop the necessary to set `owner` and `permissions` as Optional, and keep the logic of Windows inside windows_utils.rs?

@meain